src/components/dialog/Dialog.js

```diff
@@ -45,16 +45,24 @@ class Dialog extends PureComponent {
     ]);
 
     return (
-      <DialogBase className={classNames} {...restProps}>
+      <DialogBase className={classNames} {...restProps} scrollable={false}>
```

Comment:
Q: Here you're providing `scrollable={false}` but the defaultProp is true, so why not `scrollable={scrollable}`?

Reply:
A: In this case, we don't want the `DialogBase` component to be scrollable. We actually want is the `children's wrapper` to be scrollable. Therefore I'm explicitly passing `false` to `DialogBase` and let the overflow being handled by the inner Box.